src/angular-app/bellows/apps/activity/activity-app.module.ts

```diff
@@ -1,6 +1,6 @@
 import * as angular from 'angular';
-import 'angular-moment-picker/dist/angular-moment-picker.css';
-import 'angular-moment-picker/dist/angular-moment-picker.js';
+import 'angular-moment-picker/dist/angular-moment-picker.min.css';
+import 'angular-moment-picker/dist/angular-moment-picker.min.js';
```

Comment:
I don't think this is a good idea. In development, we should be importing non-minified files so that we can actually read the code we're importing. Webpack should be able to minify the fles for us, and if it can't, that's a bug (probably our config needs to be changed and then it will work).

Also, off the top of my head I don't think importing `angular-moment-picker/dist/specific-file.js` is right either. Surely we should just be doing `import * as angularMomentPicker from 'angular-moment-picker'` and let its `package.json` file declare the correct entrypoint. (I haven't looked in-depth into this one so I could be off-base here).